### ISSUE TITLE
Discrepancy between QTLJob.linkage_map_id and QTL.linkage_group.linkage_maps

### ISSUE THREAD

**Issue by Nuanda** [eSpectrum-IT/brassica#205]
When you take the 3 QTLJobs that are present in the CS data, and you will compare their `linkage_map_id` with what you get for related QTLs (you need to go through linkage_group.linkage_maps here, but they all reference a single linkage_map at the end), you will find the two linkage maps are equal in only one case:
 - BnaTNDH_01_2006a (qtl_job MapQTL_SMA_DQ06_001)

In both others QTLJob cases these maps are different:
 - BolAGDH_05_2008a vs BolAGDH_01_2002a

What should we do about that? Perhaps we can drop the QTLJob.linkage_map reference completely?

**Comment by wjurkowski** [comment]
Indeed, there is inconsistency. In qtl we have:
| QTLcarto_CIM_MB08_001  | whri_KUpE_01_LAB_GE2_2005a      | BolAGDH_01_2002a_C3  |
but in qtl_jobs:
| QTLcarto_CIM_MB08_001  | BolAGDH_05_2008a |

Removing reference will not change the fat of discrepancy in the actual data (different maps). Again, mark for curation.

**Comment by teatree1212** [comment]
issue noticed and included into curation list.

**Comment by Nuanda** [comment]
@teatree1212 Annemarie, could we "open" the curation issues that you identified as still pending, and assign them provisionally to you?

**Comment by teatree1212** [comment]
yes please.  I heard that @nowakowski also came across spelling issues etc in the database. Is there any documentation about this?
